tools/azure/createsp.sh

```diff
@@ -1,13 +1,34 @@
 #!/usr/bin/env nash
 
-if len($ARGS) != "4" {
-	echo "usage: "+$ARGS[0]+" <subscription_id>  <service_principal> <password>"
+ROLES = (
+	("Owner" "true")
+	("Reader" "true")
+)
+
+fn check_role(role) {
+	for entry in $ROLES {
+		if $entry[0] == $role {
+			return $entry[1]
+		}
+	}
+
+	return "false"
+}
+
+if len($ARGS) != "5" {
```

Comment:
If the new parameter is obligatory the docs should also be updated, I think we mention this tool there =)